group.yml

```diff
@@ -182,3 +182,14 @@ image_build_log_scanner:
   - ppc64le.log
   - task_failed.log
   - orchestrator.log
+
+# To decide whether to build and use signed RPMs, and to decide if a strict bug validation flow is necessary.
+# If it will ship via errata (state=release), we want to build signed.
+# For early access without an errata (state=pre-release), that is not required
+release_state:
+  release:
+  - x86_64
+  pre-release:
```

Comment:
`pre-release` -> `pre_release`

Reply:
Done in c53f2c14a531acae61b642fbf95c5bc56161483c